options.html

```diff
@@ -49,6 +46,11 @@
                 </div>
                 </div>
         </div>
+        <div class="row">
```

Comment:
Needed to make this change for the options window to work in iOS. It shouldn't have adverse effects on the other browsers, but we will need to check.